lesson1/homework#1/classes.js

```diff
@@ -0,0 +1,68 @@
+class Human {
+  constructor() {
+    this.name = "";
+    this.age;
+    this.dateOfBirth = "";
+  }
+  displayInfo() {
+    return `name: ${this.name};  age: ${this.age};  birthday: ${
+      this.dateOfBirth
+    };`;
+  }
+}
+
+class Employee extends Human {
+  constructor() {
+    super();
+    this.salary;
+    this.department = "";
+  }
+  displayInfo() {
+    const human = super.displayInfo();
+    console.log(
+      `${human}  department: ${this.department};  month-salary: ${
+        this.salary
+      } RUR;`
+    );
+  }
+}
+
+class Meneger extends Employee {
+  constructor() {
+    super();
+    this.managerId = "";
+    this.developers = [];
+  }
+  addDeveloper() {
+    this.developers.push(new Developer());
+  }
+  removeDeveloper(id) {
+    for (let i = 0; i < this.developers.length; i++) {
+      if (Object.keys(this.developers[i].developerId) === id) {
```

Comment:
```suggestion
      if (this.developers[i].developerId === id) {
```
`Object.keys()` возвращает массив со свойствами объекта, а тут мы передаем ему примитив (значение `developerId`), без него сравнение будет правильным:)